tests/e2e-pw/tests/templates/order-confirmation.block_theme.spec.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+const permalink = '/checkout/order-received';
+const templatePath = 'woocommerce/woocommerce//order-confirmation';
+const templateType = 'wp_template';
+
+test.describe( 'Test the order confirmation template', async () => {
+	test( 'Template can be opened in the site editor', async ( { page } ) => {
+		await page.goto( '/wp-admin/site-editor.php' );
+		await page.click( 'text=Templates' );
```

Comment:
It says "discouraged"—is that the same thing? I've copied usage from other tests and the global-setup file. I'd rather leave it as I'm just trying to get new tests added rather than update our playwright implementation.

Reply:
Yeah you're right, sorry I misread it as deprecated, I'm so used to seeing that in yellow boxes in documentation 🙈

I am ok with it being left like this for now if updating it is going to delay this PR being merged. I would like to revisit at some point though.